### ISSUE TITLE
PyLint 2.6.2 released with changelog specifying 2.7.0

### ISSUE THREAD

**Issue by jerr0328** [PyCQA/pylint#4099]
## Question
I noticed that Pylint 2.6.2 is out, but the changelog specifies changes in 2.7.0, without any mention 2.6.2. Was 2.6.2 supposed to be released as 2.7.0?

**Comment by Tenzer** [comment]
2.6.2 only have a very minimal change compared to 2.6.0: https://github.com/PyCQA/pylint/compare/pylint-2.6.0...pylint-2.6.2. It seems it was in order to fix #4097.
I'm not sure what happened with version 2.6.1 though, I guess it must have been a bad release in some way.

2.7.0 doesn't seem to have been released yet.

**Comment by Pierre-Sassoulas** [comment]
`2.6.1` was a fail, the `2.6` branch was not exactly based on pylint `2.6.0` as I assumed it was. `2.6.2` is the fix (2.6.1 rebased on the 2.6.0 tag). I released fast because when pylint latest version crash a lot of persons have broken builds  and the thumbsups on issue accumulate fast.. :) More detail on why 2.6.x version are almost empty if you're curious : #3831 

TLDR : We've released astroid 2.5 yesterday which caused the problem with 2.6 that we had to fix, and we're going to release `2.7` very soon (once we make sure there is no problem with astroid 2.5 by checking the issues that were reported today).
